pallets/capacity/src/lib.rs

```diff
@@ -121,7 +127,7 @@ pub mod pallet {
 		/// The maximum number of unlocking chunks a StakingAccountLedger can have.
 		/// It determines how many concurrent unstaked chunks may exist.
 		#[pallet::constant]
-		type MaxUnlockingChunks: Get<u32>;
+		type MaxUnlockingChunks: Get<u32> + Clone;
```

Comment:
I won't why we need a clonable type for this

Reply:
Not required. Removed